### ISSUE TITLE
How can i solve safe area issue in iPhone 12 series phone?

### ISSUE THREAD

**Issue by yogendra-leocan** [aksonov/react-native-router-flux#3732]
react-native-router-flux : 4.2.0

Get safearea issue only in iPhone 12 series phone.

Please help me.

**Comment by mtr1012** [comment]
Add to your `package.json` ` "resolutions": {
    "react-native-status-bar-height": "2.6.0"
  }`

**Comment by yogendra-leocan** [comment]
@mtr1012 I tried but not work, Can you please write all step in details?

**Comment by mtr1012** [comment]
`{
  "name": "XXX",
  "version": "0.0.1",
  "private": true,
  "scripts": {
    "android": "react-native run-android",
    "ios": "react-native run-ios",
    "start": "react-native start",
    "test": "jest",
    "lint": "eslint ."
  },
  "dependencies": {
  },
  "devDependencies": {
    "@babel/core": "^7.6.2",
    "@babel/runtime": "^7.6.2",
    "@react-native-community/eslint-config": "^0.0.5",
    "babel-jest": "^24.9.0",
    "eslint": "^6.5.1",
    "jest": "^24.9.0",
    "metro-react-native-babel-preset": "^0.56.0",
    "react-test-renderer": "16.9.0"
  },
  "jest": {
    "preset": "react-native"
  },
  "resolutions": {
    "react-native-status-bar-height": "2.6.0"
  }
}
`
Your `package.json` will look like this

**Comment by BurningFenix** [comment]
Hey Guys, anybody resolved this issue? I also added the resolutions in the package.json and still doesn't work for me.

**Comment by yogendra-leocan** [comment]
@BurningFenix Update expo cli it may solve your problem.

**Comment by nica0012** [comment]
Has this been solved?

**Comment by nica0012** [comment]
This is resolved in 4.3.0

**Comment by fekajin** [comment]
i can solved it by adding this to package.json

```
"resolutions": {
    "react-native-router-flux/**/react-navigation-stack": "^2.10.2",
    "react-native-router-flux/**/react-navigation-tabs": "^2.10.1"
  },
```
didnt try 4.0.3 version yet.

**Comment by aksonov** [comment]
Fixed in 4.3.0
